.github/workflows/blackbox.yml

```
@@ -61,20 +61,25 @@ jobs:
       - name: Prepare
         uses: ./.github/actions/prepare
 
-      - name: Start mssql
-        if: matrix.vendor == 'mssql'
-        run: |
-          curl https://packages.microsoft.com/keys/microsoft.asc | sudo tee /etc/apt/trusted.gpg.d/microsoft.asc
-          sudo add-apt-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/20.04/mssql-server-2019.list)"
-          sudo apt-get update
-          sudo apt-get install -y mssql-server
-          sudo /opt/mssql/bin/mssql-conf set network.tcpport 6105
-          sudo MSSQL_PID='express' MSSQL_SA_PASSWORD='Test@123' /opt/mssql/bin/mssql-conf -n setup accept-eula
-
       - name: Start database
-        if: matrix.vendor != 'sqlite3' && matrix.vendor != 'mssql'
+        if: matrix.vendor != 'sqlite3'
         run: docker compose -f tests/blackbox/docker-compose.yml up ${{ matrix.vendor }} -d --quiet-pull --wait
 
+      - name: MSSQL Set Compatibility Level
+        if: matrix.vendor == 'mssql'
+        run: |
+          for i in {1..30}; do
```

```suggestion
          for i in {1..30}
          do
```
pedantic : formatting across our other solutions